Modules/_zstd/compressor.c

```diff
@@ -49,98 +49,106 @@ typedef struct {
 #include "clinic/compressor.c.h"
 
 static int
-_zstd_set_c_parameters(ZstdCompressor *self, PyObject *level_or_options,
-                       const char *arg_name, const char* arg_type)
+_zstd_set_c_level(ZstdCompressor *self, int level)
 {
-    size_t zstd_ret;
-    _zstd_state* mod_state = PyType_GetModuleState(Py_TYPE(self));
-    if (mod_state == NULL) {
+    /* Set integer compression level */
+    int min_level = ZSTD_minCLevel();
+    int max_level = ZSTD_maxCLevel();
+    if (level < min_level || level > max_level) {
+        PyErr_Format(PyExc_ValueError,
+        "%d not in valid range %d <= compression level <= %d",
```

Comment:
```suggestion
        "compression level %d not in valid range %d <= level <= %d",
```

I think it is good to lead with what the value error is about.

Reply:
As above, it feels odd to me to say "compression level -2498 not in valid range MIN <= level <= MAX", I'm not a fan of the double 'level'. This may be more consistent though, and Serhiy also suggested it.

Reply:
My concern is without context the error is not clear right away (i.e. For which argument is the error being raised?)